frontend/src/test/scala/bloop/logging/RecordingLogger.scala

```diff
@@ -0,0 +1,26 @@
+package bloop.logging
+
+import scala.collection.mutable.Buffer
+
+class RecordingLogger extends AbstractLogger {
```

Comment:
Can you explain why `BufferedLogger` doesn't cut it? I don't think we need a new logger... we have plenty of them 😄

Reply:
`BufferedLogger` doesn't let us inspect the messages that it contains at the moment, but I'm happy to refactor it if you prefer.